docs/_layouts/default.html

```diff
@@ -34,6 +34,15 @@ <h2 class="project-tagline">{{ page.description | default: site.description | de
       <a href="{{ site.github.jar_url }}" class="btn">Download .jar</a>
       {% endif %}
     </header>
+    
+    <nav>
+      <a src="docs/faq.html">FAQ</a>
+      <a src="docs/modules.html">Modules</a>
+      <a src="docs/contributing.html">Contributing</a>
+      <a src="docs/troubleshooting.html">Troubleshooting</a>
+      <a src="docs/coc.html">Code of Conduct</a>
+      <a src="docs/support.html">Support</a>
```

Comment:
Issue here is the `docs/` is relative. If you're already on a page it will try to take you inside `docs/docs/`

Change it to `/docs/`